.github/workflows/release.yaml

```diff
@@ -185,7 +185,8 @@ jobs:
           version: ${{ needs.prepare.outputs.version }}
           registry: ${{ matrix.url }}
           repository: ${{ secrets[matrix.repository] }}
-          combined: true
+          platforms: "amd64, arm64, ppc64le, s390x"
+          annotation: "com.googleapis.cloudmarketplace.product.service.name=services/dynatrace-operator-dynatrace-marketplace-prod.cloudpartnerservices.goog"
```

Comment:
potentially its not hard to run same action using if `if: ${{ matrix.registry == 'gcr' }} and copy
if: ${{ matrix.registry != 'gcr' }} without annotation. So I made it pretty flexible for future, may be we need another annotations for different registries.